### ISSUE TITLE
Extension won't work if URL specifies a port number

### ISSUE THREAD

**Issue by t0pu** [tamius-han/ultrawidify#104]
**Describe the bug**
If the URL of a page a video plays on specifies a port (like `http://127.0.0.1:5500/`) ultrawidify doesn't work.
For example the "D" shortcut does nothing. 

**To Reproduce**
- Save this simple demo page as a HTML file (`uw-test.html`):
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
    <link rel="stylesheet" href="https://cdn.plyr.io/3.6.2/plyr.css" />
  </head>
  <body>
    <div>
      <video
        id="videoElement"
        src="https://upload.wikimedia.org/wikipedia/commons/transcoded/d/d7/BENNU%E2%80%99S_JOURNEY_-_NASA.webm/BENNU%E2%80%99S_JOURNEY_-_NASA.webm.720p.webm"
        muted
        autoplay
        controls
      ></video>
    </div>

    <script src="https://cdn.plyr.io/3.6.2/plyr.js"></script>
    <script> const player = new Plyr("#videoElement");</script>
  </body>
</html>
````
- fire up a http server for this file which runs on a port other that 80 or 443 (use the Live Server extension in VSCode for example)
- open the page `http://127.0.0.1:5500/uw-test.html` in the browser
- open ultrawidify settings 
- site settings > Enable extension for 127.0.0.1 > Enable
- site settings > Enable autodetection for 127.0.0.1 > Enable
- enter fullscreen
- click on the video or menu-bar to give focus to player
- press "D" to set to 21:9 mode
- watch nothing happen

- to cross check fire up the http server on port 80
- open the page `http://127.0.0.1/uw-test.html` in the browser
- do the steps above again and the "D" shortcut should set the video to 21:9 mode this time correctly

**Expected behavior**
Ultrawidify should be functional regardless if a port is specified in the URL.

**Screenshots**

![image](https://user-images.githubusercontent.com/16021175/83279990-402d3800-a1d6-11ea-9d92-0fd91f2f10b9.png)

**Browser and OS**
 * Browser/browser version: Mozilla Firefox - Version 76.0.1, Google Chrome - Version 83.0.4103.61, Microsoft Edge - Version 83.0.478.37
 * Operating system: Debian 10, Windows 10 (1909)
 * Ultrawidify version: 4.4.8

**Additional info**
The extention adds its class to the `<video>` but is seems that the css won't be loaded.

**Comment by tamius-han** [comment]
Pretty refreshing getting a bug report this well written.

I think I've fixed the problem for me. The fix will go live over the weekend for Firefox and ... well, however long Google takes with their review. Could be days, could be weeks.

In the mean time, you can test my fix: https://stuff.lionsarch.tamius.net/ultrawidify/testing/ — given that I've got no idea what's up with the bug, because it's the "it should either work for all sites or completely broken for all sites" kind of issue.

**Comment by t0pu** [comment]
I tested the version 2005.29.169 and I can now successfully switch to 21:9 mode using the "D" shortcut!
The initial problem seems to be fixed.

On the other hand while testing I noticed that the site settings won't stay on the selected ones when reopening the extension popup. I don't know if this is a "feature" when installing unpacked extensions or not, or if it's a new bug. Happens both in Firefox and Chrome.

![Peek 2020-05-30 14-18](https://user-images.githubusercontent.com/16021175/83328336-850ea880-a282-11ea-9528-d73bc9a1fc83.gif)

**Comment by tamius-han** [comment]
Well damn. 

I'll look into it this (next) week.

**Comment by tamius-han** [comment]
I honestly really need to turn off that feature where "fixes #104" auto-closes the issue for me, but oh well.

As it turns out, I was not very consistent with my usage of `window.location.host(name)`, using ~~whichever came to my mind first~~ okay let's be honest, the odds of using `.host` over `.hostname` roughly correlated with how late it was when I first wrote that bit of code and how lazy was I when I wrote that bit of code.
